test/test-layout.js

```diff
@@ -0,0 +1,43 @@
+var should = require('should');
```

Comment:
To clarify, he's been telling him that because it is codified in our style guide, which I believe we all had a chance to review.